views/form.html

```diff
@@ -18,44 +26,81 @@ <h1 class="prospect-form__heading">Contact us for FT Group Subscription options<
 	</div>
-	<div class="prospect-form o-forms">
-		<label for="prospect-form__job-title" class="prospect-form__label">Job title</label>
-		<input name="title" data-trackable="job-title" type="text" id="prospect-form__job-title" placeholder="Enter your job title" class="prospect-form__text" required></input>
-		<div class="prospect-form__errortext">Please enter your job title.</div>
+	<div class="o-forms">
+		<label for="prospect-form__job-title" class="o-forms__label">Job title</label>
+		<input name="title" data-trackable="job-title" type="text" id="prospect-form__job-title" placeholder="Enter your job title" class="o-forms__text" required>
+		<div class="o-forms__errortext">Please enter your job title.</div>
 	</div>
-	<div class="prospect-form o-forms">
-		<label for="prospect-form__company-name" class="prospect-form__label">Company name</label>
-		<input name="company" data-trackable="company-name" type="text" id="prospect-form__company-name" placeholder="Enter your company name" class="prospect-form__text" required></input>
-		<div class="prospect-form__errortext">Please enter your company name.</div>
+	<div class="o-forms">
+		<label for="prospect-form__company-name" class="o-forms__label">Company name</label>
+		<input name="company" data-trackable="company-name" type="text" id="prospect-formo__company-name" placeholder="Enter your company name" class="o-forms__text" required>
+		<div class="o-forms__errortext">Please enter your company name.</div>
 	</div>
-	<div class="prospect-form o-forms">
-		<label for="prospect-form__work-email" class="prospect-form__label">Work email address</label>
-		<input name="email" data-trackable="email-address" type="email" id="prospect-form__work-email" placeholder="Enter your work email address" class="prospect-form__text" required></input>
-		<div class="prospect-form__errortext">Please enter your work email address.</div>
+	{{#unless isUnmasking}}
+	<div class="o-forms">
+		<label for="prospect-form__work-email" class="o-forms__label">Work email address</label>
+		<input name="email" data-trackable="email-address" type="email" id="prospect-form__work-email" placeholder="Enter your work email address" class="o-forms__text" required>
+		<div class="o-forms__errortext">Please enter your work email address.</div>
 	</div>
-	<div class="prospect-form o-forms">
-		<label for="prospect-form__phone-number" class="prospect-form__label">Work phone number</label>
-		<small class="prospect-form__additional-info">5 to 15 characters (numbers only)</small>
-		<input name="phone" data-trackable="phone-number" type="text" id="prospect-form__phone-number" placeholder="Enter your phone number" class="prospect-form__text" required pattern="([0-9()+\- ])+" minlength="5" maxlength="15"></input>
-		<div class="prospect-form__errortext">Please enter a valid work phone number.</div>
+	{{/unless}}
+	<div class="o-forms">
+		{{#if isUnmasking}}
+		<label for="prospect-form__phone-number" class="o-forms__label">Phone number</label>
```

Comment:
This `if` block could just be inside the label and change the text rather than dupe the element

Reply:
Thought it was easier to read like this? Don't mind changing though

Reply:
Possibly, its just extra duplication

Reply:
👍